src/renderer/components/dock/terminal.ts

```diff
@@ -49,8 +49,8 @@ export class Terminal {
   private xterm: XTerm | null = new XTerm({
```

Comment:
Something should also change the font configs for already existing tabs if the preferences change.

Reply:
Fixed